microsoft/testsuites/mshv/test_data/README.md

```diff
@@ -0,0 +1,7 @@
+To execute this testsuite, one must provide the set of MSHV binaries to test. 
```

Comment:
Does it apply to all test cases in this mshv folder, or only added new test cases? Either way, the description should be in test suite metadata. So the generated test document contains it.

Reply:
Removed; replaced with mshv_binpath case variable.